kernel/Makefile

```diff
@@ -17,7 +17,7 @@ ccflags-y += -I$(srctree)/security/selinux -I$(srctree)/security/selinux/include
 ccflags-y += -I$(objtree)/security/selinux -include $(srctree)/include/uapi/asm-generic/errno.h
 
 ifndef KSU_MODULE
-obj-y += kernelsu.o
+obj-$(CONFIG_KSU) += kernelsu.o
```

Comment:
Maybe you can use `obj-$(CONFIG_KSU) += kernelsu.o` and then modify the build-lkm.yml?

Reply:
And then we can delete KSU_MODULE

Reply:
> Maybe you can use and then modify the build-lkm.yml?`obj-$(CONFIG_KSU) += kernelsu.o`

maybe fixed